.gitignore

```diff
@@ -20,3 +20,6 @@ build/
 # to embed ebpf programs by using go:embed, we have to copy programs first
 pkg/transparentproxy/ebpf/programs/amd64/mb_*
 pkg/transparentproxy/ebpf/programs/arm64/mb_*
+
+# Used to download manifests for k3d
+/test/k3d
```

Comment:
We don't need the `/` at the begining, right?

Reply:
it was in the original PR so I'm leaving this as is